pkg/kubernetes/scripts/virtual-machines/components/VmOverviewTabKubevirt.jsx

```diff
@@ -61,21 +61,42 @@ function getMemory(vm: Vm) {
     return _("Not Available");
 }
 
-const VmOverviewTabKubevirt = ({ vm, vmMessages }: { vm: Vm, vmMessages: VmMessages }) => {
+function getPod(vm, pods) {
+    if (!pods) {
+        return null;
+    }
+
+    const vmId = vm.metadata.uid;
+    if (!vmId) {
+        return null;
+    }
+
+    return pods.find(pod => getValueOrDefault(() => pod.metadata.labels[VM_UID_LABEL], null) === vmId);
+}
+
+const PodLink = ({ pod }) => {
+    if (!pod || !pod.metadata.namespace || !pod.metadata.name) {
+        return null;
+    }
+
+    return (<a href={`#/l/pods/${pod.metadata.namespace}/${pod.metadata.name}`}>{pod.metadata.name}</a>);
```

Comment:
What is the magic `/1` path component here?

Reply:
Actually its a letter (ascii 6C). I took this pattern from URL in browser ... Omitting that leads to `containers` view.